### ISSUE TITLE
Freecom: Sending reply from Slack is broken

### ISSUE THREAD

**Issue by jbbr** [graphcool/content#179]
Steps to reproduce:

- Start conversation on https://demo.graph.cool/freecom/
- Open corresponding channel in Slack https://freecom-team.slack.com
- Type `/fc <some message>` in slack

Expected behavior:
- Message appears on the "client"/demo page

Actual behavior:
- Error message from slackbot:
```
Darn – that slash command didn't work (error message: `400_client_error`). Manage the command at Freecom.
```

**Comment by nikolasburk** [comment]
Thanks a lot for the hint @jbbr! 🙏 We've noticed this as well, the issue apparently is due to the fact that Slack has deprecated the use of a single Slash command in a team and instead requires a "Slack app" to be installed for the same kind of functionality. We'll update the tutorial soon to make use of this new concept! Until then I added a note about it in the [Overview](https://www.graph.cool/docs/tutorials/freecom-overview-intercom-tutorial-e8a6ajt8ax/#demo:-try-out-the-final-product) chapter.
